src/modules/commentUI.js

```diff
@@ -0,0 +1,45 @@
+export default class CommentUI {
+  static addFacts = async (parentElement, facts) => {
+    parentElement.innerHTML = '';
+    const fact1 = document.createElement('p');
+    fact1.textContent = `Type: ${facts[0]}`;
+    const fact3 = document.createElement('p');
+    fact3.textContent = `Shortcut: ${facts[2]}`;
+    const fact4 = document.createElement('p');
+    fact4.textContent = `Author: ${facts[3]}`;
+
+    parentElement.appendChild(fact1);
+    parentElement.appendChild(fact3);
+    parentElement.appendChild(fact4);
+
+    return parentElement;
+  };
+
+  static showComments = (
+    ele,
+    { creation_date: date, comment: message, username: name },
+  ) => {
+    const {
+      creation_date: creationDate,
+      comment,
+      username,
+    } = { creation_date: date, comment: message, username: name };
+    const comEl = document.createElement('p');
+    comEl.innerText = `${creationDate} ${username} ${comment} `;
+    ele.appendChild(comEl);
+    return comEl;
+  };
```

Comment:
- [ ] Guys your project is looking really great, congrats! 🎉 Let's just fix an issue with the comments:

<img width="580" alt="Screen Shot 2022-07-30 at 7 43 17 PM" src="https://user-images.githubusercontent.com/83717285/182005011-dcd0a688-9117-44e4-b0f5-fd1a979fc65b.png">


The comments are stored and loaded properly, however, it's only showing the first 3 comments, if the artist has more than 3 it does not show the comment that you add:

(Also, the comment counter says NaN when you add the first comment)

<img width="634" alt="Screen Shot 2022-07-30 at 7 43 30 PM" src="https://user-images.githubusercontent.com/83717285/182004984-50df69c4-98a2-4864-9c36-4334dcd97c11.png">

Let's try to display all the comments or at least the last comments added but more than three.

Reply:
@smunozmo Hey, thanks, I've checked the project but in my local host there's no problem so please could you check again? :pray: